middle.mkd

```diff
@@ -355,6 +355,24 @@ Example of a bbox for a line crossing the date-line:
         //...
     }
 
+
+## The Poles
+
+A bounding box that contains the North Pole extends from a southwest
+corner of <minlat>°N, 180°W to a northwest corner of 90°N, 180°E. Viewed
+on a globe, this bounding box approximates a spherical cap.
+
+    "bbox": [-180.0, <minlat>, 180.0, 90.0]
+
+A bounding box that contains the South Pole extends from a southwest
+corner of 90°S, 180°W to a northwest corner of <maxlat>°S, 180°E.
+
+    "bbox": [-180.0, <minlat>, 180.0, 90.0]
```

Comment:
[-180, -90, 180, <maxlat> ]

Also note that the degrees symbol (°) is not permitted in RFCs.  That's a relic of their ASCII heritage.  You might want to use * instead.